pkgs/development/libraries/qt-5/modules/qtsystems.nix

```diff
@@ -0,0 +1,54 @@
+{ qtModule
+, stdenv
+, lib
+, bluez
+, libevdev
+, libX11
+, pkg-config
+, qtbase
+, udev
+, wrapQtAppsHook
+}:
+
+qtModule {
+  pname = "qtsystems";
+
+  outputs = [
+    "out"
+    "dev"
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [
+    "bin"
+  ];
+
+  qtInputs = [
+    qtbase
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapQtAppsHook
+  ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [
+    bluez
+    libevdev
+    libX11
+    udev
+  ];
+
+  qmakeFlags = [
+    "CONFIG+=git_build"
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [
+    "CONFIG+=ofono"
+    "CONFIG+=udisks"
+    "CONFIG+=upower"
+  ];
+
+  postFixup = lib.optionalString stdenv.hostPlatform.isLinux ''
+    wrapQtApp $bin/bin/servicefw
+  '';
+
+  meta = {
+    maintainers = with lib.maintainers; [ OPNA2608 ];
```

Comment:
Shouldn't we also set at least license and description?

Reply:
`meta` attributes are set by `qtModule`, the majority of Qt modules don't even have a custom `meta`.

https://github.com/NixOS/nixpkgs/blob/884e3b68be02ff9d61a042bc9bd9dd2a358f95da/pkgs/development/libraries/qt-5/qtModule.nix#L70-L76

I'm just overriding `meta.maintainers` because this evidently isn't a commonly-used & updated module, and it will be used by afew packages I'm working on adding down the line.